hassio/src/ingress-view/hassio-ingress-view.ts

```diff
@@ -102,14 +99,14 @@ class HassioIngressView extends LitElement {
         }
         if (!addonInfo.version) {
           await showAlertDialog(this, {
-            text: this.supervisor.localize("my.error_addon_not_installed"),
+            text: this.hass.localize("supervisor.my.error_addon_not_installed"),
             title: addonInfo.name,
           });
           await nextRender();
           navigate(`/hassio/addon/${addonInfo.slug}/info`, { replace: true });
         } else if (!addonInfo.ingress) {
           await showAlertDialog(this, {
-            text: this.supervisor.localize("my.error_addon_no_ingress"),
+            text: this.hass.localize("supervisor.my.error_addon_no_ingress"),
```

Comment:
Add a fallback for the translations, as we now use the HA translations, older HA versions will not have them.